typesense/src/main/resources/reference.conf

```diff
@@ -0,0 +1,10 @@
+alpakka.typesense {
+  host = "http:localhost:8108"
+  api-key = "Hu52dwsas2AdxdE"
```

Comment:
These values should be left blank to force users to set them instead.
Add `src/test/resources/application.conf` to set these for testing.